### PR TITLE
Minimalistic Backplate Mod for V0.0 and V0.1

### PR DESCRIPTION
**Pull request by khay8485**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
I added a minimalistic backplate for V0.0 and V0.1 to allow the printchamber to be closed while the electronics are open.
This will allow cooler motors (V0.0 extruder is hot hot hot) and electronics (pi @ V0.0, pi and mcu @ V0.1)
#### How was it tested? 
I printed and mounted it on my untuned V0 hence the spooky first layer on the pictures. Gave the files to two other V0 owners and they didn´t complained about errors.
#### Any background context you want to provide?
Everything is written above, i think.
#### Screenshots (if appropriate)
Pictures are in the provided files, and i have no idea how to add them directly here.
#### Further notes
Have a nice day. :)

### CODE REVIEW

**Comment by khay8485** [comment]
I read this as i will have to modify the top-level readme, but i think i should not do that before the mod is merged?
```
The PR modifies the top-level README.md of the category folder adding the
contribution to the table. Read the top part of the file for instructions on how
to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
to fill out the compatibility matrix to indicate which versions of the Voron printer
the submission is compatible with.
```

**Comment by khay8485** [comment]
> 
> 
> I read this as i will have to modify the top-level readme, but i think i should not do that before the mod is merged?
> 
> ```
> The PR modifies the top-level README.md of the category folder adding the
> contribution to the table. Read the top part of the file for instructions on how
> to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
> to fill out the compatibility matrix to indicate which versions of the Voron printer
> the submission is compatible with.
> ```

i may have understood it now... ;)

File: printer_mods/khay/micro_backplate/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Scrolling through discord, there a V0 owners like me that experienced really hot electronics or motors because of the enclosed back of the printer.
+As a serious aduld without kids or furry kids, you could leave the backplate off the printer to allow for airflow for the electronics/motors. 
+Unfortunately this will result in loosing hot air from the printchamber. 
+So in addition to Weaslus "Electronics-Bay-Separator" or the V0.1 revision motor panel i have sketched this "Micro Backplate" out of the original
```
Comment (KiloQubit) [inline printer_mods/khay/micro_backplate/README.md:4]: Capitalize `I` in `I have sketched`

File: printer_mods/khay/micro_backplate/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+corners for the backplate. It is split in two parts to fit on a spec V0. 
+
```
Comment (KiloQubit) [inline printer_mods/khay/micro_backplate/README.md:6]: Include your image in the README, like this

    Backplate

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please remove the ZIP file and upload individual files (look at other mods for examples)

Also see README comments below.

File: printer_mods/khay/micro_backplate/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Scrolling through discord, there a V0 owners like me that experienced really hot electronics or motors because of the enclosed back of the printer.
```
[the remark this answers is not in the document]
Comment (khay8485) [reply printer_mods/khay/micro_backplate/README.md]: done

File: printer_mods/khay/micro_backplate/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+As a serious aduld without kids or furry kids, you could leave the backplate off the printer to allow for airflow for the electronics/motors.
```
[the remark this answers is not in the document]
Comment (khay8485) [reply printer_mods/khay/micro_backplate/README.md]: done

File: printer_mods/khay/micro_backplate/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Scrolling through discord, there a V0 owners like me that experienced really hot electronics or motors because of the enclosed back of the printer.
+As a serious aduld without kids or furry kids, you could leave the backplate off the printer to allow for airflow for the electronics/motors. 
+Unfortunately this will result in loosing hot air from the printchamber.
```
[the remark this answers is not in the document]
Comment (khay8485) [reply printer_mods/khay/micro_backplate/README.md]: done

File: printer_mods/khay/micro_backplate/README.md (hunk 1 above)
Comment (khay8485) [reply printer_mods/khay/micro_backplate/README.md]: done

File: printer_mods/khay/micro_backplate/README.md (hunk 2 above)
Comment (khay8485) [reply printer_mods/khay/micro_backplate/README.md:6]: done

**Comment by khay8485** [comment]
> 
> 
> Thanks for your mod!
> 
> Please remove the ZIP file and upload individual files (look at other mods for examples)
> 
> Also see README comments below.

Have done that too. I am embarrassed about all that typos...

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates!

I see that you have added more images, can you include all of them in the README?

**Comment by khay8485** [comment]
> 
> 
> Thanks for the updates!
> 
> I see that you have added more images, can you include all of them in the README?

I am slowly getting used to git. Some more changes? I´m feeling challenged to learn. ;-)

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**